ironfish/src/storage/levelup/transaction.ts

```diff
@@ -27,23 +29,35 @@ export class LevelupTransaction implements IDatabaseTransaction {
   waitingResolve: (() => void) | null = null
   id = 0
 
+  public lockAcquisition: Meter
+  public lockContention: Meter
+  public lockWaitTime: number
   static id = 0
 
   constructor(db: LevelupDatabase) {
     this.db = db
     this.id = ++LevelupTransaction.id
     this.batch = new LevelupBatch(db)
+    this.lockAcquisition = new Meter()
```

Comment:
Because these meters don't persist beyond a single transaction I don't think these Meters are useful. Aren't they just always going to be a Meter with `.add(1)` called one time? Because a transaction will only attempt to acquire a lock once.